docs/source/API/core/view/layoutLeft.rst

```diff
@@ -0,0 +1,92 @@
+``LayoutLeft``
+==============
+
+.. role:: cppkokkoskokkos(code)
+   :language: cppkokkoskokkos
+
+Header File: ``Kokkos_Layout.hpp``
+
+Usage:
+
+.. code-block:: cppkokkos
```

Comment:
TODO: check if using `cppkokkos` (instead of `cpp`) makes sense here.

Reply:
no, cppkokkos should NOT be used for code blocks